qa/mixed-cluster/build.gradle

```diff
@@ -63,5 +63,19 @@ BuildParams.bwcVersions.withWireCompatible { bwcVersion, baseName ->
     tasks.register(bwcTaskName(bwcVersion)) {
       dependsOn "${baseName}#mixedClusterTest"
     }
+
+    tasks.named("${baseName}#mixedClusterTest").configure {
```

Comment:
I think we should use skip versions instead? I think there are other qa tests that reuse the rest api spec tests too.
This should work, since we test new behaviour and not old behavior.

Reply:
We should also ensure that at least a base percentile and percentile_ranks tests are ran in mixed cluster tests.

Reply:
Looks like we're doing both. I saw a number of ml tests that are using `skip` in the YAML. We should align on one or the other.